docs/src/app/components/markdown/lists-markdown.md

```diff
@@ -0,0 +1,31 @@
+# Headline 1
+
+This is a paragraph describing something.
+
+## Subheadline
+
+Detailes description here. And some code:
+```javascript
```

Comment:
If it works best we can do so! Both (js and javascript) are possible with highlight.js